src/Microsoft.Maintainability.Analyzers/UnitTests/RemoveUnusedLocalsTests.cs

```diff
@@ -14,7 +19,54 @@ protected override DiagnosticAnalyzer GetBasicDiagnosticAnalyzer()
 
         protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
         {
-            return new CSharpRemoveUnusedLocalsAnalyzer();
+            throw new NotImplementedException();
+        }
+
+        [Fact]
```

Comment:
Can you add some more tests: local accessed only in lambda, local passed by ref or out?